test/reporters/dot.spec.js

```diff
@@ -13,6 +13,10 @@ describe('Dot reporter', function() {
   var useColors;
   var windowWidth;
 
+  function color(type, str) {
+    return '\u001b[' + Base.colors[type] + 'm' + str + '\u001b[0m';
+  }
+
```

Comment:
For simplicity of comparison, it might be easier _not_ to be so literal.
```js
function speedString(type, str) {
  return '##' + type + '-' + str;
};

Base.color = speedString;
```